packages/frontend/src/components/SchedulersView/SchedulersView.tsx

```diff
@@ -0,0 +1,252 @@
+                                        <Text fz="xs" key={i}>
+                                            {channel}
+                                        </Text>
+                                    ))}
+                                >
+                                    <Text fz="xs" color="gray.6" underline>
+                                        Slack
+                                    </Text>
+                                </Tooltip>
+                            )}
+                        </Group>
+                    );
+                },
+            },
+            {
+                id: 'frequency',
+                label: 'Frequency',
+                cell: (item) => {
+                    return (
+                        <Text fz="xs" color="gray.6">
+                            {getHumanReadableCronExpression(item.cron)}
+                        </Text>
+                    );
+                },
+                meta: { style: { width: 200 } },
+            },
+            {
+                id: 'lastDelivery',
+                label: 'Last delivery start',
+                cell: (item) => {
+                    const currentLogs = logs.filter(
+                        (log) => log.schedulerUuid === item.schedulerUuid,
+                    );
+                    return currentLogs.length > 0 ? (
+                        <Group spacing="sm">
+                            <Text fz="xs" color="gray.6">
+                                {formatTime(currentLogs[0].createdAt)}
+                            </Text>
+                            {getLogStatusIcon(currentLogs[0], theme)}
+                        </Group>
+                    ) : (
+                        <Text fz="xs" color="gray.6">
+                            No deliveries started
+                        </Text>
+                    );
+                },
+            },
+            {
+                id: 'actions',
+                cell: (item) => {
+                    return (
+                        <Box
+                            component="div"
+                            onClick={(e) => {
+                                e.stopPropagation();
+                                e.preventDefault();
+                            }}
+                        >
+                            <SchedulersViewActionMenu
+                                item={item}
+                                projectUuid={projectUuid}
+                            />
+                        </Box>
+                    );
+                },
+                meta: {
+                    style: { width: '1px' },
+                },
+            },
+        ],
+        [users, charts, dashboards, projectUuid, logs, theme],
+    );
+
+    return (
+        <Table className={classes.root} highlightOnHover>
+            <thead>
+                <tr>
+                    {columns.map((column) => {
+                        return (
```

Comment:
you can use inline function without `return (` to remove extra indentation since this is a very simple rendering function without extra logic